src/wmnds/patterns/footer/_footer.scss

```diff
@@ -21,4 +21,169 @@
       display: block;
     }
   }
+
+  // footer v2 only styles
+  &--v2 {
+    padding: $size-lg $size-xs;
+
+    .wmnds-btn {
+      display: block;
+      color: get-color(primary);
+      background-color: $white;
+    }
+
+    .wmnds-footer__link {
+      color: currentColor;
+      font-weight: normal;
+      text-decoration: none;
+
+      &:hover {
+        color: currentColor;
+        text-decoration: underline;
+      }
+    }
+  }
+
+  @media screen and (max-width: $breakpoint-md - 1) {
+    .wmnds-footer__updates-col {
+      padding-bottom: $size-md * 1.25;
+
+      &:not(:last-child) {
+        margin-bottom: $size-xsm;
+        border-bottom: 1px solid get-color(secondary, 70);
+      }
+
+      p {
+        font-size: 0.937rem;
+      }
+    }
+  }
+
+  .wmnds-footer__heading {
+    margin-bottom: $size-md * 1.25;
+    padding-bottom: $size-md * 1.125;
+    border-bottom: 1px solid get-color(secondary, 70);
+
+    @media screen and (max-width: $breakpoint-md - 1) {
+      margin-bottom: $size-md;
+      font-weight: normal;
+
+      &.wmnds-collapse-heading {
+        margin: 0;
+        padding: $size-xsm 0;
+        border: 0;
+        font-family: $x-font-family;
+        font-size: $size-md * 1.125;
```

Comment:
There is a scss function called type-setting, see here for how it is used: https://github.com/wmcadigital/wmn-design-system/blob/master/src/wmnds/assets/sass/_typography.scss

...this is what it does in the background:
https://github.com/wmcadigital/wmn-design-system/blob/master/src/wmnds/assets/sass/_mixins.scss

Implement the size that you are after.

The sizes and type headings have been done in a way so that they all follow a very particular vertical rhythm. Hence the no "multiplying vars" etc.